.gitpod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+# See here all gitpod images available: https://hub.docker.com/r/gitpod/workspace-python-3.10/tags
+# Current python version: 3.10.5
+FROM gitpod/workspace-python-3.10@sha256:8a7ad4f0bbaa281a36cf2a87b772354638a14d7383f0a755b9ea32596ee99632
```

Comment:
Let's use 3.9. It's the latest Lambda runtime supports, we can upgrade to 3.10 when they do. Also deeply appreciate the security mindset in using an immutable image